bench_method.go

```diff
@@ -38,7 +38,24 @@ type peerTransport struct {
 // benchmarkCaller exposes method to dispatch requests for benchmark.
 type benchmarkCaller interface {
 	// Call dispatches a request using the provided transport.
-	Call(transport.Transport) (time.Duration, error)
+	Call(transport.Transport) (benchmarkCallResult, error)
+}
+
+type benchmarkCallResult interface {
+	// Latency returns the time taken to send request and receive response.
+	Latency() time.Duration
+}
+
+type benchmarkCallLatencyResult struct {
+	latency time.Duration
+}
+
+func (r benchmarkCallLatencyResult) Latency() time.Duration {
+	return r.latency
+}
+
+func newBenchmarkCallLatencyResult(latency time.Duration) benchmarkCallLatencyResult {
```

Comment:
This function should be above the `Latency` one - see https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering for function and group ordering